game/world/worldobjects.cpp

```diff
@@ -322,6 +322,7 @@ Npc* WorldObjects::addNpc(size_t npcInstance, const Vec3& pos) {
   Npc* npc   = new Npc(owner,npcInstance,pstr);
   npc->setPosition  (pos.x,pos.y,pos.z);
   npc->updateTransform();
+  npc->autoEquipWeapons();
```

Comment:
Another override for `WorldObjects::addNpc(size_t npcInstance, std::string_view at)` seem to be missing.

Reply:
I left it out because this function calls `restPositionToTA` where it's already done. You want to have it added for consistency with other `addNpc` function?

Additionally I found that`B_RefreshAtInsert` is called as well and added it.

Reply:
> You want to have it added for consistency with other addNpc function?

Ideally, I think, we need to make it robust and leave no room for inconsistency. After giving more thought to it: maybe it's even better to add `ProcessPolicy` as a parameter to `npc` constructor. This will make any explicit/implicit use of `isPlayer` correct.